### ISSUE TITLE
Patients showing up in wrong username's register in tablet and admin

### ISSUE THREAD

**Issue by space-ravi** [kamal1978/LTFHC#303]
Issue is on **production** app and admin site. 

Patients are showing up in the wrong username's register in both tablet and admin. Reminder that all registers should be showing up based on username. 

For example, the Mtoa June 2020 tablet (username cmulibinge) shows patients from Undugu (username bkasanga) and Kataki (username ekalwele). 

Additional info from field team: 
- LTFHC reported that Mtoa never uploaded their June data, so it is odd that the admin website shows a register/report for Mtoa in June 2020. 
- LTFHC reported that they uploaded data from Undugu for May/June. They never saw it on the website, but noticed it showed up instead in the Mtoa register. 

Note that the problems started happening after the servers were down at the end of May (see email from Ravi to Kamal on May 26-27). Not sure if this is related but wanted to share this info.

**Comment by space-ravi** [comment]
Update: I emailed files from the affected users. I renamed the files to correspond to the relevant health center. 

The questions we want to answer is: 
- Is there an issue causing patients from one username to show up in another username's profile? For example, they mentioned patients from Mtoa were showing up in the Undugu tablet.
- Is it OK for the health workers to upload data to the server?

Note that I should receive the Mtoa db file shortly.
